### DISCUSSION TITLE
Force settings for my users

### DISCUSSION THREAD

**Discussion by mikymick** [2fast-team/2fast#128] [Q&A] [answered]
Hello,

First : great product, thank you.

I'm trying a bunch of 2FA apps for some of my company users that won't use their phones for 2FA.

I like very much that a password is needed to access the generated T-OTP.

Id' like to go further in security, and be able to "force" some settings such as : "auto logout : ON + auto lock after 10 mn inactivity"

I've located settings.dat but can't edit it as it seems partially crypted.

One way to do it (botched) would be to replace the settings.dat with my carefully tuned version through company GPO, so every now and the, the app when relaunched would use the company settings.

But that wouldn't prevent the user from unticking settings for their convenience (and lowering the security) or even to reset to factory default then reimport their 2fast container file afterwards.

Is there a way I could disable these settings from being modified by the end user ?

Thanks in advance for your inputs.
Mike.

**Comment by jp-weber** [comment, marked as answer]
Hello,

currently, it is not possible to disable these settings for end users. In a next version, I will implement that the app can be installed via MDM with a configuration based on: https://learn.microsoft.com/en-us/windows/client-management/mdm/enterprisemodernappmanagement-csp

However, the enterprise function and other functionalities (categories and import backups from other apps) are only available to users/devices via a subscription or one-time purchase.

**Comment by jp-weber** [comment]
The newest preview version is now supporting MDM. The enterprise functions are in the beta free to use. I would be delighted if you would test the beta version and also submit feedback. What I'm currently implementing is the custom background setting for the login pages so that the corporate identity can flow in.
https://github.com/2fast-team/2fast/releases/tag/v1.3.0-beta
